docs/changelog.rst

```diff
@@ -1,6 +1,31 @@
-*********
 Changelog
-*********
+=========
+
+.. changelog template:
+
+  .. _release-X.Y.Z:
+
+  X.Y.Z (YYYY-MM-DD)
+  ------------------
+
+  New features:
+
+  Bug fixes:
+
+  Documentation:
+
+  For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
+
+.. _release 2.1.0:
+
+2.1.0 (2025-08-06)
```

Comment:
```suggestion
2.1.0 (2025-08-11)
```

Even if we merge today let's do that

Reply:
Fixed: https://github.com/canonical/craft-grammar/pull/108/commits/493f9fccbda38120fedb7613760a6d97af3b304f